### PR TITLE
move teambuff to team overview

### PR DESCRIPTION
**Pull request by frzyc**
## Describe your changes
Move the content of the teambuff tab to the team setting modal, which now serves as a "team overview"
image
Since this is the entry point towards engaging with the teams section, users are aware of teambuffs that interact with loadouts, and at a glance, users can also get a general idea if their loadouts are correct.

 - Resolves https://github.com/frzyc/genshin-optimizer/issues/1606
## Checklist before requesting a review (leave this PR as draft if any part of this list is not done.)

- [ ] I have commented my code in hard-to understand areas.
- [ ] I have made corresponding changes to README or wiki.
- [ ] For front-end changes, I have updated the corresponding English translations.
- [ ] I have run `yarn run mini-ci` locally to validate format and lint.
- [ ] If I have added a new library or app, I have updated the deployment scripts to ignore changes as needed

### CODE REVIEW

**Comment by nguyentvan7** [comment]
Why was the outer card removed for teammates? The floating effect looks nice/new on desktop, but it makes the UI really hard to parse on mobile
Screenshot_20240308_075257_Chrome

**Comment by frzyc** [comment]
> Why was the outer card removed for teammates? The floating effect looks nice/new on desktop, but it makes the UI really hard to parse on mobile 

Because there are now 4 characters in a 5 column layout, I thought the character card was using up too much padding. There is not much we can do with mobile other than creating separate UI just for mobile, which is something I don't really want to undertake until the UI around this area hits maturity (don't want to spend time on mobile layout optimization when we havent finalized the desktop UI)

**Comment by nguyentvan7** [comment]
Also I really dislike the tall Add Character button. Can we make it a fixed height instead of 100% height

**Comment by StainAE86** [comment]
> Also I really dislike the tall Add Character button. Can we make it a fixed height instead of 100% height

Agreed

**Review by nguyentvan7 — APPROVED** [review]
Still ambivalent but approving to unblock
